lib/rules/time-min-milliseconds/__tests__/index.js

```diff
@@ -0,0 +1,211 @@
+"use strict"
+
+const messages = require("..").messages
+const ruleName = require("..").ruleName
+const rules = require("../../../rules")
+
+const rule = rules[ruleName]
+
+const MIN_VALUE = 100
+
+testRule(rule, {
+  ruleName,
+  config: [MIN_VALUE],
+
+  accept: [ {
+    code: "a { transition-delay: 0.1s; }",
+  }, {
+    code: "a { transition-delay: 0.2s; }",
+  }, {
+    code: "a { -webkit-transition-delay: 0.2s; }",
+  }, {
+    code: "a { tRaNsItIoN-dElAy: 0.2s; }",
+  }, {
+    code: "a { TRANSITION-DELAY: 0.2s; }",
+  }, {
+    code: "a { transition-delay: 3s; }",
+  }, {
+    code: "a { transition-delay: 100ms; }",
+  }, {
+    code: "a { transition-delay: 200ms; }",
+  }, {
+    code: "a { transition-delay: 5000ms; }",
+  }, {
+    code: "a { transition-duration: 0.15s; }",
+  }, {
+    code: "a { -webkit-transition-duration: 0.15s; }",
+  }, {
+    code: "a { transition-duration: 2.8s; }",
+  }, {
+    code: "a { transition-duration: 330ms; }",
+  }, {
+    code: "a { transition-duration: 4700ms; }",
+  }, {
+    code: "a { transition: foo 0.8s linear; }",
+  }, {
+    code: "a { -webkit-transition: foo 0.8s linear; }",
+  }, {
+    code: "a { tRaNsItIoN: foo 0.8s linear; }",
+  }, {
+    code: "a { TRANSITION: foo 0.8s linear; }",
+  }, {
+    code: "a { transition: foo 0.8s 200ms ease-in-out; }",
+  }, {
+    code: "a { animation-delay: 0.2s; }",
+  }, {
+    code: "a { -webkit-animation-delay: 0.2s; }",
+  }, {
+    code: "a { animation-delay: 3s; }",
+  }, {
+    code: "a { animation-delay: 200ms; }",
+  }, {
+    code: "a { animation-delay: 5000ms; }",
+  }, {
+    code: "a { animation-duration: 0.15s; }",
+  }, {
+    code: "a { -webkitanimation-duration: 0.15s; }",
+  }, {
+    code: "a { animation-duration: 2.8s; }",
+  }, {
+    code: "a { animation-duration: 330ms; }",
+  }, {
+    code: "a { animation-duration: 4700ms; }",
+  }, {
+    code: "a { animation: foo 0.8s linear; }",
+  }, {
+    code: "a { -webkit-animation: foo 0.8s linear; }",
+  }, {
+    code: "a { animation: foo 0.8s 200ms ease-in-out; }",
+  }, {
+    code: "a { animation-delay: -2.5s; }",
```

Comment:
Good question! I think we should check only positive values.

According to specs `transition-duration` and `animation-duration` can't be negative. Negative values can be in shorthand properties `transition` and `animation`, but it's hard to parse what time is which in shorthands.

Also, negative delay is a rare case. Developer could set negative value in two cases: intentionally for some reason, or by mistake.

What do you think?